src/schema/me/__tests__/credit_cards.test.ts

```diff
@@ -18,7 +18,7 @@ describe("CreditCards", () => {
     const query = gql`
       {
         me {
-          creditCards(first: 1) {
+          creditCards(first: 1, limit: 1) {
```

Comment:
I don't follow this example. Why do you need a `limit` in addition to `first`? Aren't they the same?